cli/cmd/logout.go

```diff
@@ -41,7 +41,7 @@ func newCmdLogout() *cobra.Command {
 }
 
 func runLogout(_ context.Context, cmd *cobra.Command) error {
-	err := auth.Logout()
+	err := team.Logout()
```

Comment:
I think `Logout` should be under auth, not under `team`

Reply:
`auth` in the current repository or the `cloudquery-api-go`? I resisted putting it in the `cloudquery-api-go` on the basis that it felt more like a [user](https://github.com/cloudquery/cloudquery/blob/main/cli/internal/auth/logout.go) of the shared code e.g. it defines what keys to store.